service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -87,6 +86,8 @@
     private String publicKey;
     @Value("${liqpay.private.key}")
     private String privateKey;
+    private static final Integer bibBagsCapacity = 120;
+    public static final String langCode = "ua";
```

Comment:
according to convention, constants should have name like: LANG_CODE, BAGS_CAPACITY etc